README.md

```diff
@@ -1,37 +1,26 @@
-## Welcome to GitHub Pages
```

Comment:
I think we need to clearly define the purpose of the project first in the readme as well as in the project description which shows above.

Later we can define problems we solve and our approach.